_layouts/email.html

```diff
@@ -51,8 +51,8 @@ <h2>{{ page.name }}</h2>
         );
         const recipients = `{{ page.recipients | join: ',' }}`;
         const cc = `{{ page.cc | join: ',' }}`;
-        const ccText = cc !== null && cc.length > 0 ? `cc=${cc}&` : ``;
-        location.href = `mailto:${recipients}?${ccText}subject=${subject}&body=${body}`;
```

Comment:
Do we want to even support cc either? Should both recipients and cc all just be mashed into bcc?

Reply:
I was wondering that... Only concern I guess is some of the templates clearly address certain people and not the folks who are listed as CC. Thoughts?

Reply:
Well your email client would tell you that you got bcc'ed. So if we move all recipients to bcc, and we kept cc's... it wouldn't be that useful to know you got cc'ed if you don't know who the recipients are by email anyways.

By that logic, we ought to bcc everyone if we are going to bcc the main recipients.

Reply:
@bingbongle rebased this to latest master (w/gatsby) - any further thoughts on it?

Reply:
@avimoondra your logic makes sense. I think everyone should be on `bcc`.

Reply:
Cool @dionlarson @bingbongle i'll update to merge cc and bcc. we can at a future time, migrate the templates to not support any key other than recipients.